composer.json

```diff
@@ -25,8 +25,8 @@
     "description" : "Common libraries for Phiremock",
     "license" : "GPL-3.0-or-later",
     "require" : {
-        "php" : "^7.2|^8.0",
-        "laminas/laminas-diactoros": "^2.0",
+        "php" : "^8.1",
```

Comment:
I think this change is not needed, the previous version condition already accepted php8.1.

Reply:
Fixed